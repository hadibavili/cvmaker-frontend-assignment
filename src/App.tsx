import Countries from "pages/Countries/Countries";

const App = () => {
   return <Countries />;
};
export default App;
