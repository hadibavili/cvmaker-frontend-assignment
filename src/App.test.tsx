import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
import renderer from "react-test-renderer";
import App from "./App";

const client = new ApolloClient({
   cache: new InMemoryCache(),
   uri: process.env.REACT_APP_API_URL,
});

describe("App", () => {
   it("App snapshot testing", () => {
      const countries = renderer
         .create(
            <ApolloProvider client={client}>
               <App />
            </ApolloProvider>
         )
         .toJSON();
      expect(countries).toMatchSnapshot();
   });
});
