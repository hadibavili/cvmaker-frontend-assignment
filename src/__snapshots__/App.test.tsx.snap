// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App snapshot testing 1`] = `
Array [
  <div
    className="sc-kDTinF jeEOrV"
  >
    <label
      className="sc-eCImPb dpOmTe"
    >
      keyword
    </label>
    <input
      className="sc-hKwDye eKRolE"
      onChange={[Function]}
      placeholder="Example: Ne"
      value=""
    />
    <button
      className="sc-bdvvtL gnuMZn"
      onClick={[Function]}
    >
      Search
    </button>
  </div>,
  <div
    className="sc-iCfMLu fbnDEN"
    data-testid="spinner"
  >
    <svg
      className="spinner"
      viewBox="0 0 50 50"
    >
      <circle
        className="path"
        cx={25}
        cy={25}
        fill="none"
        r={20}
        strokeWidth={5}
      />
    </svg>
  </div>,
  <div
    className="sc-iqseJM EDwOO"
  />,
]
`;
